src/pages/404.tsx

```diff
@@ -1,43 +1,47 @@
-import { graphql } from 'gatsby';
+import { StaticQuery, graphql } from 'gatsby';
 import React from 'react';
 import Hero from '../components/hero';
 import Layout from '../components/layout';
 import Navigation from '../components/navigation';
 import { LearnPageData } from '../types';
 import { findActive } from '../util/findActive';
 
-type Props = {
-  data: LearnPageData;
-};
-
-export default ({ data }: Props) => {
+export default () => {
   // Get current page related data.
-  const { navigationSections } = findActive(data.sections.group);
-
   const title = `PAGE NOT FOUND`;
   const description = `You've hit a route that does not exist.`;
 
   return (
-    <Layout title={title} description={description}>
-      <Hero title={title} />
-      <Navigation sections={navigationSections} />
-      <article className="article-reader">
-        <h1 className="article-reader__headline">{title}</h1>
-        <div>
-          <p>
-            The page you're trying to access does not exist. Go back to the
-            Homepage or find what you're looking for in the menu.
-          </p>
-          <p>
-            Take me back to the <a href="/">Homepage</a> →
-          </p>
-        </div>
-      </article>
-    </Layout>
+    <StaticQuery
+      query={query}
+      render={data => {
```

Comment:
nit: Retain the type.

```suggestion
      render={(data: LearnPageData) => {
```

Reply:
Thanks @sagirk!  Retained the type. New to Typescript so unaware of using types.